qiling/os/windows/utils.py

```diff
@@ -46,3 +46,11 @@ def is_file_library(string):
 
 def string_to_hex(string):
     return ":".join("{:02x}".format(ord(c)) for c in string)
+
+
+def find_size_function(ql, func_addr):
+    # We have to retrieve the return address position
+    code = ql.mem.read(func_addr, 0x100)
+    opcodes = [b"\xc3", b"\xc2", b"\xcb", b"\xca"]
```

Comment:
opcode sounds funny. these are  interrupt handler ? or any proper name ?

Reply:
https://c9x.me/x86/html/file_module_x86_id_280.html
I think the name is correct

Reply:
opcode is a type. but these opcode design for a purpose.

Reply:
I mean, what i'm trying to do with this function is to find the return opcode.
Maybe return_opcodes is a better name? Or as you wish, i don't really mind

Reply:
Return from Procedure. we need a shorter name

Reply:
from wikipedia
```
In computing, an opcode (abbreviated from operation code, also known as instruction machine code, instruction code, instruction syllable, instruction parcel or opstring) is the portion of a machine language instruction that specifies the operation to be performed
```